docs/pages/versions/unversioned/sdk/screen-orientation.md

```diff
@@ -54,224 +55,14 @@ Tick the `Requires Full Screen` checkbox in Xcode. It should be located under `P
-
-### `ScreenOrientation.SizeClassIOS`
-
-Each iOS device has a default set of [size classes](https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/TheAdaptiveModel.html) that you can use as a guide when designing your interface.
-
-- **`SizeClassIOS.REGULAR`**
-- **`SizeClassIOS.COMPACT`**
-- **`SizeClassIOS.UNKNOWN`**
-
-### `ScreenOrientation.WebOrientationLock`
-
-An enum representing the lock policies that can be applied on the web platform, modelled after the [W3C specification](https://w3c.github.io/screen-orientation/#dom-orientationlocktype). These values can be applied through the [`lockPlatformAsync`](#screenorientationlockplatformasyncplatforminfo) method.
-
-- **`PORTRAIT_PRIMARY`**
-- **`PORTRAIT_SECONDARY`**
-- **`PORTRAIT`**
-- **`LANDSCAPE_PRIMARY`**
-- **`LANDSCAPE_SECONDARY`**
-- **`LANDSCAPE`**
-- **`ANY`**
-- **`NATURAL`**
-- **`UNKNOWN`**
-
-## Object Types
-
-### `ScreenOrientation.PlatformOrientationInfo`
-
-- **screenOrientationConstantAndroid (_integer_)**: A constant to set using the Android native [API](https://developer.android.com/reference/android/R.attr.html#screenOrientation). For example, in order to set the lock policy to [unspecified](https://developer.android.com/reference/android/content/pm/ActivityInfo.html#SCREEN_ORIENTATION_UNSPECIFIED), -1 should be passed in. (**Android only**)
-- **screenOrientationArrayIOS (Array[Orientation])**: An array of orientations to allow on the iOS platform (**iOS only**)
-- **screenOrientationLockWeb (_WebOrientationLock_)**: A web orientation lock to apply in the browser (**web only**)
-
-### `ScreenOrientation.ScreenOrientationInfo`
-
-- **orientation (_Orientation_)**: The current orientation of the device
-- **verticalSizeClass (_SizeClassIOS_)**: The [vertical size class](https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/TheAdaptiveModel.html) of the device (**iOS only**)
-- **horizontalSizeClass (_SizeClassIOS_)**: The [horizontal size class](https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/TheAdaptiveModel.html) of the device (**iOS only**)
-
-### `ScreenOrientation.OrientationChangeEvent`
-
-- **orientationLock (_OrientationLock_)**: The current OrientationLock of the device.
-- **orientationInfo (_ScreenOrientationInfo_)**: The current ScreenOrientationInfo of the device.
-
-### `Subscription`
-
-A [subscription object](https://github.com/expo/expo/blob/master/packages/%40unimodules/react-native-adapter/src/EventEmitter.ts#L13).
-
-## Function Types
-
-### `ScreenOrientation.OrientationChangeListener`
-
-#### Args
-
-- **event (_OrientationChangeEvent_)**: An update with the most recent OrientationChangeEvent.
-
-#### Returns
-
-`void`
+<APISection packageName="expo-screen-orientation" apiName="ScreenOrientation" />
 
 ## Error Codes
```

Comment:
Are we going to handle this section via the autogenerated docs tool as well? 🤔

Reply:
Unfortunately "Error Codes" information are not exposed to the TS anywhere, this applies to all of the packages, so at this moment of time we need to stick to manually adding those entries in. 

I will take another look at this problem again after the migration of all modules and issues related to that would be fixed. 🙂